src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/Shared/BaseAS1000.js

```diff
@@ -927,13 +927,25 @@ class AS1000_PFD_AirportInfos extends NavSystemElement {
 }
 class AS1000_PFD_AirspeedReference {
     constructor(_valueElem, _statusElem, _refSpeed, _displayName) {
-        this.isDisplayed = false;
+        this._isDisplayed = PersistVar.get(`.VSRef.${_displayName}.IsDisplayed`, false);
```

Comment:
This line has the one small quibble I have remaining.  I like the dot-notation for namespacing, but do you think it would be better to just append that automatically to the ATC MODEL in the persistence handler so that you don't need to with that first dot every time you're manipulating a variable?  If there's a reason you went with this way instead I'm not opposed to it, mostly curious from an ease-of-use perspective.